libkbfs/root_metadata.go

```diff
@@ -916,7 +916,36 @@ func (rmds *RootMetadataSigned) IsLastModifiedBy(
 	return nil
 }
 
-// DecodeRootMetadataSigned deserializes a metaddata block into the specified versioned structure.
+// DecodeRootMetadata deserializes a metadata block into the specified
```

Comment:
I couldn't figure out an easy way to factor out the common code with `DecodeRootMetadataSigned` below.